### PR TITLE
Add Xdebug recipe

### PR DESCRIPTION
**Pull request by texnixe**

### CODE REVIEW

**Comment by texnixe** [comment]
Let me know what you think and how to improve it.

File: content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+(image: debug-session.png)
+
+In this first step, the `$notes` and `$tag` variable are still uninitialized. 
+
+(screenshot: uninitialized-variables.png)
+
+If we now click on `Step over` in the debug toolbar (or press <kbd>F10</kbd>), we can see how the content in the debug panel changes. Now the `$tag` variable is no longer uninitialized, but changes to an empty string.
+
+If we use `Step over` again,  we will now see how the `$notes` variable has changed:
```
Comment (lukasbestle) [inline content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt]: It looks like there was supposed to be a screenshot here that's missing.

**Review by lukasbestle — COMMENTED** [review]
Awesome and very practical!

Because my own setup works a bit differently, I didn't test it myself. But I had some issues before with a setup that uses `start_with_request = yes`, particularly that Xdebug started to complain that it couldn't connect to the debugger when VS Code or the debug extension were not running. Have you tested what happens if VS Code is quit? Does the server still work fine in `debug` mode without any warnings?

I've also proofread the recipe and made some changes directly to avoid additional review loops. I hope that's OK for you.

**Comment by texnixe** [comment]
Hey @lukasbestle Thanks a lot for proofreading! I'll double-check your concerns and the missing screenshot later.

File: content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt [hunk 2]
```diff
@@ -189,6 +189,8 @@ In this first step, the `$notes` and `$tag` variable are still uninitialized.
 
 If we now click on `Step over` in the debug toolbar (or press <kbd>F10</kbd>), we can see how the content in the debug panel changes. Now the `$tag` variable is no longer uninitialized, but changes to an empty string.
 
+(screenshot: tag-variable.png)
```
Comment (lukasbestle) [inline content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt]: Shouldn't the screenshot be below the description?

Comment (texnixe) [reply content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt]: 🙈 😢

Comment (texnixe) [reply content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt]: If it's still messed up now, I'll just go to bed.

**Review by lukasbestle — APPROVED** [review]
In `notes-variable.png`, there's already the `desert` tag visible. But TBH nobody is going to notice, so feel free to merge the recipe. It is really great now!

**Pull request merged**